distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -2005,6 +1770,7 @@ public void testCancelDownload() throws Exception {
                 filename,
                 "Resource retrieval cancelled. ",
                 "STOPPED");
+
```

Comment:
For the above `verifyNotification` calls, we probably should get the last message returned instead of hard-coding the indices.

Also, I'm surprised that the activity verification doesn't fail since there's no guarantee that the cancellation message will be the second one (index 1) all the time.